h2o-py/tests/testdir_algos/automl/pyunit_automl_xgboost_gblinear_large.py

```diff
@@ -0,0 +1,128 @@
+        ]), dict(name="GBM", steps=[
+            dict(id="def_5", group=1, weight=10),
+            dict(id="def_2", group=2, weight=10),
+            dict(id="def_3", group=2, weight=10),
+            dict(id="def_4", group=2, weight=10),
+            dict(id="def_1", group=3, weight=10),
+            dict(id="grid_1", group=4, weight=60),
+            dict(id="lr_annealing", group=7, weight=10),
+        ]), dict(name="DeepLearning", steps=[
+            dict(id="def_1", group=3, weight=10),
+            dict(id="grid_1", group=4, weight=30),
+            dict(id="grid_2", group=5, weight=30),
+            dict(id="grid_3", group=5, weight=30),
+        ]), dict(name="completion", steps=[
+            dict(id="resume_best_grids", group=6, weight=60),
+        ]), dict(name="StackedEnsemble", steps=[
+            dict(id="monotonic", group=9, weight=10),
+            dict(id="best_of_family_xglm", group=10, weight=10),
+            dict(id="all_xglm", group=10, weight=10),
+        ])])
+    aml2.train(y=ds.target, training_frame=ds.train)
+    print(aml2.leaderboard)
+    for m in aml2.leaderboard.as_data_frame(use_pandas=False)[1:]:
+        assert model_is_in_automl(h2o.get_model(m[0]), aml)
+
+    aml_with_gblinear = H2OAutoML(max_models=MAX_MODELS, seed=1, modeling_plan=[
+        dict(name="XGBoost", steps=[
+            dict(id="def_2", group=1, weight=10),
+            dict(id="def_1", group=2, weight=10),
+            dict(id="def_3", group=3, weight=10),
+            dict(id="grid_1", group=4, weight=90),
+            dict(id="grid_gblinear", group=4, weight=90),  # << XGBoost GBLinear booster grid
+            dict(id="lr_search", group=7, weight=30),
+        ]), dict(name="GLM", steps=[
+            dict(id="def_1", group=1, weight=10),
+        ]), dict(name="DRF", steps=[
+            dict(id="def_1", group=2, weight=10),
+            dict(id="XRT", group=3, weight=10),
+        ]), dict(name="GBM", steps=[
+            dict(id="def_5", group=1, weight=10),
+            dict(id="def_2", group=2, weight=10),
+            dict(id="def_3", group=2, weight=10),
+            dict(id="def_4", group=2, weight=10),
+            dict(id="def_1", group=3, weight=10),
+            dict(id="grid_1", group=4, weight=60),
+            dict(id="lr_annealing", group=7, weight=10),
+        ]), dict(name="DeepLearning", steps=[
+            dict(id="def_1", group=3, weight=10),
+            dict(id="grid_1", group=4, weight=30),
+            dict(id="grid_2", group=5, weight=30),
+            dict(id="grid_3", group=5, weight=30),
+        ]), dict(name="completion", steps=[
+            dict(id="resume_best_grids", group=6, weight=60),
+        ]), dict(name="StackedEnsemble", steps=[
+            dict(id="monotonic", group=9, weight=10),
+            dict(id="best_of_family_xglm", group=10, weight=10),
+            dict(id="all_xglm", group=10, weight=10),
+        ])])
+    aml_with_gblinear.train(y=ds.target, training_frame=ds.train)
```

Comment:
Example of how to run normal automl with gblinear grid.

Reply:
is this what we want to give to the customer when they want to try gblinear?
I mean if it's one single customer, I think it's fine, but it fixes AutoML behavior once and for all…